packages/mui-joy/src/Link/Link.test.tsx

```diff
@@ -72,7 +72,7 @@ describe('<Link />', () => {
 
       events.forEach((n) => {
         const event = n.charAt(2).toLowerCase() + n.slice(3);
```

Comment:
```suggestion
        const event = n.charAt(2).toLowerCase() + n.slice(3)  as keyof typeof fireEvent;
```
And then remove `any` from line below.

`keyof typeof fireEvent` is basically similar to `EventType` and both are exported from `test/utils/index.ts`.